users/serializers.py

```diff
@@ -22,6 +27,10 @@ class Meta:
 class UserPrivateSerializer(UserPublicSerializer):
     username = serializers.CharField(read_only=True)
     id = serializers.IntegerField(read_only=True)
+    profile_photo = Base64ImageField(
```

Comment:
No need to duplicate it here as long as we have this field in the parent serializer (also for the id field)

Reply:
Removed